source/fe/fe_system.cc

```diff
@@ -1204,8 +1118,28 @@ compute_fill (const Mapping<dim,spacedim>                      &mapping,
                           fe_data.update_flags);
 
 
-  if (flags & (update_values | update_gradients))
+  if (flags & (update_values | update_gradients | update_hessians))
     {
+      if (fe_data.is_first_cell())
+        {
+          // Initialize the internal FE objects for the base elements
+          for (unsigned int base_no=0; base_no<this->n_base_elements(); ++base_no)
```

Comment:
This block no longer compiles on master because `is_first_cell` has been removed in #1316. The whole block you have here was removed in #1315. I think all you need to do to make this run is to just remove the entire `if` block, but I'm running this through the tester and will report back.